index.js

```diff
@@ -27,16 +27,13 @@ class Plugin extends PluginMiniAccounts {
     this._account = opts.account
     this._db = opts.db || 'machinomy_db'
     this._provider = opts.provider || 'http://localhost:8545'
-    this._minimumChannelAmount = new BigNumber(opts.minimumChannelAmount || 100)
+    this._minimumChannelAmount = new EthUnit(opts.minimumChannelAmount || 1, 'gwei')
```

Comment:
If this is the smallest divisible unit, shouldn't the channel capacity be more than 1?

Reply:
:+1: